spec/notifications/new_logbook_entry_created_notification_spec.rb

```diff
@@ -0,0 +1,28 @@
+require "rails_helper"
+
+RSpec.describe(NewLogbookEntryCreatedNotification, type: :model) do
+  let(:logbook_entry) { create(:logbook_entry) }
+
+  context "when logbook entries are created" do
+    it "creates database notifications" do
+      # Two notifications expected because a User must exist for a Logbook_Entry to be created
+      expect { logbook_entry }.to(change(Notification, :count).by(2))
+    end
+
+    it "creates database notifications of the right type" do
+      logbook_entry
```

Comment:
Is this needed?

Reply:
I vaguely remember having to touch the record for RSpec to execute the insert in the db, is that the case here?

Reply:
That's correct. Looks awkward to me but :man_shrugging: